### PR TITLE
PR: Migrate blog

### LINKED ISSUE #4: Migrate blogs to new engine

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes #4

### CODE REVIEW

File: models/blog-post.ini [hunk 1]
```diff
@@ -13,6 +13,20 @@ label = Author
 type = string
 width = 1/2
 
+[fields.tags]
+label = Tags
+type = checkboxes
+choices = iPython, Modes, Interpreter, PyQt, PySide, Winpython, Py3, v2-1-14, v2-3, release, v2-1, release, Features, iPython, v0-11, v2-3-2, release, Windows
+
```
Comment (goanpeca) [inline models/blog-post.ini]: Lets add a desctiption that explains to the user what he/she needs to do if a new category or a new tag is needed (pointing to the right place)

File: models/blog-post.ini [hunk 2]
```diff
@@ -13,6 +13,20 @@ label = Author
...
+[fields.category]
+label = Category
+type = select
+choices = Releases, Spyder
+
```
Comment (goanpeca) [inline models/blog-post.ini:21]: Lets add descriptions to this, maybe use the example of the link of wordpress, to explain the difference between tag and category

**Comment by goanpeca** [comment]
@rlaverde any input on the comments?

**Comment by goanpeca** [comment]
LGTM, merge?

**Comment by rlaverde** [comment]
> @rlaverde any input on the comments?

I just pushed two commits that resolve the comments

> LGTM, merge?

yes

**Pull request merged**